docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1407,10 +1408,12 @@ to_timestamp(expression)
 
 ### `to_timestamp_millis`
 
-Converts a value to RFC3339 millisecond timestamp format (`YYYY-MM-DDT00:00:00.000Z`).
-Supports timestamp, integer, and unsigned integer types as input.
-Integers and unsigned integers are parsed as Unix nanosecond timestamps and
```

Comment:
this text incorrectly implies numeric arguments are interpreted as `nanoseconds` which is not the case (they are treated as `milliseconds). 

I also tried to clean up the language about what a "unix timetamp" meant (really means seconds from unix epoch)